release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -533,10 +603,55 @@ This bug fix ensures Kubernetes is used as the source of truth. OVN-Kubernetes n
+
+*Builds*
+
+* Previously, Dockerfile builds could not change permissions of `/etc/pki/ca-trust` or create files inside it. This issue was caused by fixing link:https://bugzilla.redhat.com/show_bug.cgi?id=1826183[*BZ#1826183*] in version 4.6, which added support for HTTPS proxies with CAs for builds and ALWAYS mounted `/etc/pki/ca-trust`. This prevented builds that included their own CAs or modified the system trust store from working correctly at runtime. The current release fixes this issue by reverting Bug 1826183. Now, builder images that include their own CAs work again. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1891759[*BZ#1891759*])
+
+* Previously,  after upgrading from {product-title} version 4.5 to version 4.6, trying to git clone from a private repository failed because builds did not add proxy information to the git configuration that was used to pull the source code. As a result, the source code could not be pulled if the cluster used a global proxy and the source was pulled from a private git repository. The current release fixes this issue: It corrects how git is configured when the cluster uses a global proxy. Now, performing git clone can pull source code from a private git repository if the cluster uses a global proxy. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1896446[*BZ#1896446*])
+
+* Previously, the node pull secret feature did not work: Node pull secrets were not used if `forcePull: true` was set in the Source and Docker strategy builds. As a result, builds failed to pull images that required the cluster-wide pull secret. The current release fixes this issue. Now node pull secrets are always merged with user-provided pull secrets. As a result, builds can pull images when `forcePull: true` is set and the source registry requires the cluster-wide pull secret. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1883803[*BZ#1883803*])
+
+* Previously, OpenShift builds failed on Git clone when SCP-style ssh locations were specified. This was caused by Golang URL parsing, which does not accommodate Git SCP-styled ssh locations. As a result, OpenShift builds and Source-to-Image (S2I) failed when those types of source URLs were supplied. The current release fixes this issue: It augments the builds and S2I to bypass Golang URL parsing and strip the `ssh://` prefix. Now, OpenShift Builds and S2I accommodate Git SCP-styled ssh locations (link:https://bugzilla.redhat.com/show_bug.cgi?id=1884270[*BZ#1884270*])
```

Comment:
```suggestion
* Previously, {product-title} builds failed on `git clone` when SCP-style SSH locations were specified because of Golang URL parsing, which does not accommodate Git SCP-styled SSH locations. As a result, {product-title} builds and Source-to-Image (S2I) failed when those types of source URLs were supplied. Now, builds and S2I bypass Golang URL parsing and strip the `ssh://` prefix to accommodate Git SCP-styled SSH locations (link:https://bugzilla.redhat.com/show_bug.cgi?id=1884270[*BZ#1884270*])
```